src/types/VibeBaseButtonComponentProps.ts

```diff
@@ -0,0 +1,15 @@
+import React, { MouseEventHandler } from "react";
+import VibeComponentProps from "./VibeComponentProps";
+
+export default interface VibeBaseButtonComponentProps<HTMLElementType = HTMLButtonElement> extends VibeComponentProps {
+  /** Callback function to run when the button is clicked */
+  onClick?: MouseEventHandler<HTMLElementType>;
+  /** aria controls - receives id for the controlled region */
+  ariaControls?: string;
+  /** Whether the button should be disabled or not */
+  disabled?: boolean;
+  /** aria to be set if the popup is open */
+  ariaHasPopup?: React.HTMLProps<HTMLElementType>["aria-haspopup"];
+  /** aria to be set if the popup is open */
+  ariaExpanded?: boolean;
```

Comment:
making sure - do both `ariaHasPopup` and `ariaExpanded` have the same description?
If so, should they always have the same value?
And, if so, why do we need both of them at the same time?

Reply:
I Admit I copy paste old description which was not accurate, will be fixed